src/services/map-events-mananger/consts/pickOptions.enum.ts

```diff
@@ -1,12 +1,12 @@
 /**
  *  NO_PICK,    - will not pick entities
  *  PICK_FIRST  - first entity will be picked . use Cesium.scene.pick()
- *  PICK_ONE    - in case a few entities are picked plonter is resolved . use Cesium.scene.drillPick()
  *  PICK_ALL    - all entities are picked. use Cesium.scene.drillPick()
+ *  MULTI_PICK  - every selected entity is picked. If chosen twice will be unselected. use Cesium.scene.pick()
  */
 export enum PickOptions {
 	NO_PICK,
 	PICK_FIRST,
-	PICK_ONE,
-	PICK_ALL
+	PICK_ALL,
```

Comment:
why pick_one was removed? 
pick_one and pick_first have different meanings .
also can remove multi_pick